common/src/main/java/com/graphicsfuzz/common/transformreduce/GlslShaderJob.java

```diff
@@ -152,8 +175,12 @@ public void makeUniformBindings() {
     // Add bindings to any uniforms not referenced in the shaders, so that we don't end up in
     // a situation where some uniforms are unbound.
     for (String uniformName : getPipelineInfo().getUniformNames()) {
-      if (!getPipelineInfo().hasBinding(uniformName)) {
-        getPipelineInfo().addUniformBinding(uniformName, nextBinding++);
+      if (!getPipelineInfo().hasBindingOrIsPushConstant(uniformName)) {
+        if (pushConstant.isPresent() && pushConstant.get().equals(uniformName)) {
+          getPipelineInfo().addUniformBinding(uniformName, true, 0);
```

Comment:
Comment on the fact that we pass 0 as a dummy binding number, which will be ignored.

Reply:
done